tests/upgrades/test_virtwho.py

```diff
@@ -84,18 +83,19 @@ def test_pre_create_virt_who_configuration(
         )
         default_loc = target_sat.api.Location(id=default_loc_id).read()
         org = target_sat.api.Organization(name=ORG_DATA['name']).create()
-        default_loc.organization.append(entities.Organization(id=org.id))
+        default_loc.organization.append(target_sat.api.Organization(id=org.id))
         default_loc.update(['organization'])
+        org.sca_disable()
```

Comment:
In 6.12 or below versions SCA is disabled already, so I think it is not required, @yanpliu can you please check and update?

Reply:
@Gauravtalreja1 Yes , you are right, this update is not for satellite6.12 , it is for satellite6.13 . Satellite6.12 is SCA disabled by default